backend/server/tests/getCurrentTask.test.ts

```diff
@@ -51,8 +51,8 @@ describe('Test getCurrentTask', () => {
     expect(currentTask1.status.state).toBe(StateItem.PENDING);
 
     const currentTask2 = getCurrentTask(2, challenges);
-    expect(currentTask2.id).toBe(7);
-    expect(currentTask2.description).toBe('Eat your breakfast in bed');
+    expect(currentTask2.id).toBe(6);
```

Comment:
Consider please `toEqual` with some object shape https://jestjs.io/docs/en/using-matchers 
I suppose it's better to have only one expect per test.